src/Symfony/Component/Cache/Traits/PdoTrait.php

```diff
@@ -53,7 +53,7 @@ private function init($connOrDsn, $namespace, $defaultLifetime, array $options)
         } elseif (is_string($connOrDsn)) {
             $this->dsn = $connOrDsn;
         } else {
-            throw new InvalidArgumentException(sprintf('"%s" requires PDO or Doctrine\DBAL\Connection instance or DSN string as first argument, "%s" given.', __CLASS__, is_object($connOrDsn) ? get_class($connOrDsn) : gettype($connOrDsn)));
+            throw new InvalidArgumentException(sprintf('"%s" requires a PDO or a Doctrine\DBAL\Connection instance or a DSN string as first argument, "%s" given.', __CLASS__, is_object($connOrDsn) ? get_class($connOrDsn) : gettype($connOrDsn)));
```

Comment:
would be `a PDO instance or a Doctrine\DBAL\Connection instance`, or `a PDO or Doctrine\DBAL\Connection instance`

Reply:
Fixed in both cache and store service